src/components/Extension/PrivateExtensions/actions/IntakeScheduling/api.service.ts

```diff
@@ -153,5 +200,16 @@ export const bookAppointment = async ({
     return jsonRes
   } catch (error) {
     throw error
+  } finally {
+    const timeEnd = new Date().valueOf()
+    log(
+      {
+        message: `Book Appointment: ${timeEnd - timeStart} milliseconds`,
```

Comment:
I don't know what message you used for on the server-side but assuming they're distinctive enough to easily know what is what.